en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store needs an application that captures online orders. For each order, the store also needs to generate a notification for the sales manager as well as update the inventory records.
```

Comment:
shall we change `an online shoe store needs` to `an online shoe store uses`

Reply:
shall we change `the store also needs` to `the application also needs`

Reply:
shall we change `inventory records.` to `inventory records and notify the status to the client who places the order.`

Reply:
Done